minecraft_servers/lymmzy/manifest.json

```diff
@@ -9,7 +9,8 @@
      "en"
   ],
   "social": {
-    "discord": "https://discord.gg/tc7XBYY4eH"
+    "discord": "https://discord.gg/tc7XBYY4eH",
+    "web": "https://shop.lymmzy.com"
```

Comment:
You can add this with the `"shop"` key. For the "website"-key, we only accept website with server information/presentation, no shop-sites.